From 62f51c0e8fc8ddb1a2a64adcef13796db6bc7b1b Mon Sep 17 00:00:00 2001
From: Xinyu Chen <xinyu.chen@freescale.com>
Date: Thu, 8 Nov 2012 12:06:00 +0800
Subject: [PATCH 096/602] ENGR00232884 imx6: add ntfs-3g utility into package

Add ntfs-3g for mount.
Add ntfsfix for fsck and filesystem detection.

Change-Id: Ia2ffb44f14006e6d78602c323eab016763a1b6de
Signed-off-by: Xinyu Chen <xinyu.chen@freescale.com>
---
 imx6/imx6.mk | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/imx6/imx6.mk b/imx6/imx6.mk
index 43e36db..941f88c 100644
--- a/imx6/imx6.mk
+++ b/imx6/imx6.mk
@@ -248,6 +248,11 @@ PRODUCT_PACKAGES += \
 	libext2_uuid	\
 	libext2fs
 
+# ntfs-3g binary
+PRODUCT_PACKAGES += \
+	ntfs-3g		\
+	ntfsfix
+
 # for CtsVerifier
 PRODUCT_PACKAGES += \
     com.android.future.usb.accessory
-- 
1.8.0

